documentation/deleteblock_block1.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="132" height="34.5"><g transform="translate(0, 0)"><g transform="scale(1.5, 1.5)"><path d="m0.5 8.5 L 0.5 4.5 A 4 4 90 0 1 4.5 0.5 L 8.5 0.5 L 8.5 2.5 L 18.5 2.5 L 18.5 0.5 L 79.5 0.5 L 83.5 0.5 A 4 4 90 0 1 87.5 4.5 L 87.5 8.5 L 83.5 8.5 L 83.5 5.5 L 79.5 5.5 L 79.5 15.5 L 83.5 15.5 L 83.5 12.5 L 87.5 12.5 L 87.5 16.5 A 4 4 90 0 1 83.5 20.5 L 79.5 20.5 L 18.5 20.5 L 17.5 20.5 L 17.5 22.5 L 9.5 22.5 L 9.5 20.5 L 8.5 20.5 L 4.5 20.5 A 4 4 90 0 1 0.5 16.5 L 0.5 12.5 L 0.5 8.5 z" style="fill:#C4C4C4;fill-opacity:1;stroke:#A0A0A0;stroke-width:1;stroke-linecap:round;stroke-opacity:1;" /><text style="font-size:10px;fill:#282828;font-family:sans-serif;text-anchor:end"><tspan x="77" y="14">delete block</tspan></text><text style="font-size:6.666666666666667px;fill:#282828;font-family:sans-serif;text-anchor:end"><tspan x="77" y="12"></tspan></text></g></g></svg>
```

Comment:
What is this file used for?

Reply:
It was the former image for the delete block, should I delete it?

Reply:
Please delete it.